test_unstructured_ingest/dest/azure-cognitive-search.sh

```diff
@@ -79,7 +80,11 @@ PYTHONPATH=${PYTHONPATH:-.} "$RUN_SCRIPT" \
   --reprocess \
   --input-path example-docs/fake-memo.pdf \
   --work-dir "$WORK_DIR" \
+  --chunking-strategy by_title \
+  --chunk-combine-text-under-n-chars 150 --chunk-new-after-n-chars 1500 --chunk-max-characters 2500 --chunk-multipage-sections --chunk-no-include-orig-elements \
```

Comment:
Maintain readability
```suggestion
  --chunk-combine-text-under-n-chars 150 \
  --chunk-new-after-n-chars 1500 \
  --chunk-max-characters 2500 \
  --chunk-multipage-sections \
  --chunk-no-include-orig-elements \
```

Reply:
Agreed, tidied with: [a line per test arg for better readability](https://github.com/Unstructured-IO/unstructured/pull/3311/commits/89cf90a71d618e19e9e261879b82744c8d2f3bc5)